### PR TITLE
update tools-overview.md for deployment/publishing

### PR DESCRIPTION
**Pull request by mdrichardson**
Suggested changes to the tools-overview readme:

* Remove the "install bot extension" line since it is no longer needed in the updated Azure CLI
* Change the "download" and "publish" links to point to the official docs since they currently point nowhere and can cause some confusion. Alternatively, we could paste all of the commands in here, but since they don't use `msbot` I didn't think that was necessary

### CODE REVIEW

File: tools-overview.md [hunk 1]
```diff
@@ -267,14 +267,10 @@ The Bot Framework Emulator is a desktop application that allows bot developers t
 
 # Publish
 
-You can use the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) tool to [create](#create-azure-bot-service-bot), [download](#download-azure-bot-service-bot) and [publish](#publish-azure-bot-service-bot) your bot to Azure Bot Service.
+You can use the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) tool to [create](#create-azure-bot-service-bot), [download](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-deploy-az-cli?view=azure-bot-service-4.0&tabs=csharp#publish-your-bot-to-azure) and [publish](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-deploy-az-cli?view=azure-bot-service-4.0&tabs=csharp#publish-your-bot-to-azure) your bot to Azure Bot Service.
```
Comment (stevengum) [inline tools-overview.md]: Is there a download section in the docs? Your links for [download] and [publish] are the same.

Comment (mdrichardson) [reply tools-overview.md]:
```suggestion
You can use the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) tool to [create](#create-azure-bot-service-bot), [download](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-deploy-az-cli?view=azure-bot-service-4.0&tabs=csharp#download-the-bot-from-azure) and [publish](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-deploy-az-cli?view=azure-bot-service-4.0&tabs=csharp#publish-your-bot-to-azure) your bot to Azure Bot Service.
```

There are. Looks like I missed a copy/paste

**Comment by mdrichardson** [comment]
That was intentional since there is a "create" section in that markdown file. There weren't sections for download or publish, which is why I pointed them to the docs, instead.

I think it would be better to have them all point to the docs, to minimize places where things need to be updated, but that would also require removing additional sections from the markdown file. I can edit the request to reflect that, but want to make sure that you want to go that direction, first.

**Comment by stevengum** [comment]
I’d rather have as much as possible point to the docs because (in my experience) we’re better about keeping those maintained than the “docs” here.

@cleemullins, @kaiqb, @sgellock do you agree?

**Comment by sgellock** [comment]
@stevengum the more we can point to articles and data linked to the docs, the more we can ensure we have all things pointed to a single set of topics that can be maintained by the docs team.  so yes, I agree with you

**Comment by Kaiqb** [comment]
@sgellock @stevengum @cleemullins - this content) is already in the [docs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tools?view=azure-bot-service-4.0).

**Comment by stevengum** [comment]
@mdrichardson can you rebase off of master and then we'll get this merged in? Thanks!

**Pull request merged**